python/kserve/kserve/model_server.py

```diff
@@ -322,7 +391,7 @@ async def servers_task():
                 servers.append(self._grpc_server.start(self.max_threads))
             await asyncio.gather(*servers)
 
-        asyncio.run(servers_task())
+        await servers_task()
```

Comment:
Where are we creating the event loop ?

Reply:
> This update makes a minor modification to the ModelServer code, modifying the start() function to be async. This was necessary in order for tests of the gRPC server functionality to run. The impact upon users of kserve is that any code that uses the ModelServer must either:

>await ModelServer.start() if in an async context, or
wrap ModelServer.start() with asyncio.run() in a sync context.
Otherwise, all functionality is preserved.

read your comment. but, wouldn't be a breaking change for the custom model users ?